### PR TITLE
Connect store to API

### PR DESCRIPTION
**Pull request by deyemiobaa**
#### Connect bookstore to API using redux-thunk

- Use redux-thunk
- Dispatch API calls and update server and UI

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.8.1",
```
Comment (descholar-ceo) [inline package.json:6]: - [ ] `redux-thunk` was supposed to have been added to the project as a package and used as a middleware <= Would you add it?

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @deyemiobaa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (deyemiobaa) [reply package.json:6]: Hi, @descholar-ceo. As seen in my package.json file. I have installed the new reduxJS toolkit instead. This lets me have access to redux-thunk and apply middleware without installing those packages independently

Comment (descholar-ceo) [reply package.json:6]: It should be used as a middleware then in the store

Comment (deyemiobaa) [reply package.json:6]: @descholar-ceo 
I have access to this by default using the reduxjs toolkit. it's redundant to install them again
[see here](https://redux.js.org/usage/configuring-your-store#simplifying-setup-with-redux-toolkit)
image

**Review by Meltrust — APPROVED** [review]
_Note:  Redux toolkit is now allowed schoolwide, and documentation is being updated.  Approving as indicated._

Hi @deyemiobaa,


## Wow, you did it 🎉
Brilliant
## Thank you for implementing the changes requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (descholar-ceo) [reply package.json:6]: I see your point, your project should be ready then!

**Comment by deyemiobaa** [comment]
Thank you, @Meltrust.

**Pull request merged**